public/src/components/calls/calls_page_parent.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import { Row, Col } from 'react-bootstrap';
+import CallsSideBar from './calls_sidebar';
+import CallsPageMain from './calls_page_main';
+
+export default class CallPage extends Component {
+  componentDidMount() {
+    console.log('#############meow', this.props);
+    // const { current_campaign,
+    //         has_user_joined_campaign,
+    //         calls_made,
+    //         next_call,
+    //         current_call,
+    //         script_questions,
+    //         current_script } = this.props;
+  }
+
+  render() {
+    const { current_campaign,
+            has_user_joined_campaign,
+            calls_made,
+            next_call,
+            current_call,
+            script_questions,
+            current_script } = this.props;
+    return (
+      <Row className="show-grid">
+        <Col xs={5} md={4} lg={3}>
+          <CallsSideBar
+            current_campaign={current_campaign}
+            joined_campaign={has_user_joined_campaign}
+            calls_made={calls_made}
+          />
+        </Col>
+        <Col xs={7} md={8} lg={9}>
+          <CallsPageMain
+            current_campaign={current_campaign}
```

Comment:
Few questions:
- Will `CallsPageMain` be the form component. In that case, should we call it `CallsPageForm` to be specific?
- Should we pass in the form name to both components?